presto-docs/src/main/sphinx/release/release-0.281.rst

```diff
@@ -0,0 +1,58 @@
+=============
+Release 0.281
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix timeout for optimization rules.
+* Improve memory accounting for rehashing.
+* Improve outer join where the join key is from the output of another outer join which can have many null values. The optimization is controlled by session property ``randomize_outer_join_null_key_strategy``.
+* Add ability to express concurrency levels as a function of number of available workers.
+* Add an optimization rule for queries which have multiple similar aggregations, with and without filters. The optimization is controlled by session parameter ``merge_aggs_with_and_without_filter``.
+* Add an optimization rule to merge aggregations when there are multiple duplicate aggregations in the aggregation node. The optimization is controlled by session property ``merge_duplicate_aggregations``.
+* Add :func:`f_cdf()` and :func:`inverse_f_cdf()` functions.
+* Add :func:`gamma_cdf()`` and :func:`inverse_gamma_cdf()` functions.
```

Comment:
@rohanpednekar please check if there is corresponding documentation for these functions